packages/postgraphile-core/__tests__/integration/__snapshots__/schema.test.js.snap

```diff
@@ -1228,6 +1238,15 @@ type UpdatePersonPayload {
 
 # A universally unique identifier as defined by [RFC 4122](https://tools.ietf.org/html/rfc4122).
 scalar UUID
+
+type WrappedUrl {
+  url: NotNullUrl!
+}
+
+# An input for mutations affecting \`WrappedUrl\`
+input WrappedUrlInput {
+  url: NotNullUrl
```

Comment:
shouldn’t this one be not null too?

Reply:
Yes if it doesn’t have a default 👍

Reply:
Ah you gave it a default, but set the default to null. I don’t really see this as a bug; but please remove “default null” and see if that fixes it. Otherwise there’s probably another nullableIf we need to track down.

Reply:
I gave a `default null` to the `site b.wrapped_url` not the url inside. so `site` inside the `Person` & `PersonInput` is correctly nullable. but the `url` inside `WrappedUrlInput` should not be. and indeed if I replace `site b.wrapped_url default null` with `site b.wrapped_url not null` it does not affect the `WrappedUrlInput` type

Reply:
Ah, my bad 👍